oex_plugin/openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -4,12 +4,128 @@
 
 from django.utils.translation import gettext_lazy as _
 
+from openedx.core.djangoapps.user_api import errors
+from openedx.core.djangoapps.user_api.errors import (
+    AccountUpdateError,
+    AccountValidationError,
+    PreferenceValidationError
+)
```

Comment:
why do you import the module and then import separate parts of the same module?

Reply:
Agree! I didn’t notice that it was already there since it was taken from the platform.